pkg/dockerregistry/server/admin.go

```diff
@@ -81,9 +84,14 @@ func (bh *blobHandler) Delete(w http.ResponseWriter, req *http.Request) {
 		return
 	}
 
+	err := bh.Cache.Remove(bh.Digest)
+	if err != nil {
+		dcontext.GetLogger(bh).Errorf("blobHandler: ignore error: unable to remove %q from cache: %v", bh.Digest, err)
```

Comment:
is this going to log an error in cases where the digest simply didn't exist in the cache?

Reply:
@legionus please be sure to answer this question ^^

Reply:
@bparees the current cache implementation returns an error only when the digest is invalid, which should never happen.

Reply:
thanks